### PR TITLE
array_str

### LINKED ISSUE #11961: array_str
working on (#8431-> #17801)

### PR DESCRIPTION
**Pull request by Sarvesh-Kesharwani**
Closed #11961

### CODE REVIEW

**Comment by hmahmood24** [comment]
Hey @Sarvesh-Kesharwani, can you please run `pre-commit` to resolve the formatting of your PR. Also, please run your test locally first via `pytest` and attach the screenshots here so I can see the test logs. Thanks

**Comment by Sarvesh-Kesharwani** [comment]
[array_str_logs.txt](https://github.com/unifyai/ivy/files/12014071/array_str_logs.txt)
pycharm64_K6YJcrnJH5

Hey @hmahmood24 , while testing for some input tensors jax is returning "<U5" dtype tensors, which
ivy's test_frontend_function functions is not able to handle, could you please help here, 
i have attached the traces returned.
thank you

**Comment by Sarvesh-Kesharwani** [comment]
waiting for review.

**Review by abdulasiraj — CHANGES_REQUESTED** [review]
E   ivy.utils.exceptions.IvyException: <U27 is not supported by ivy
E   Falsifying example: test_jax_numpy_array_str(
E       frontend='jax',
E       on_device='cpu',
E       dtype_and_x=(['int32'], [array([0], dtype=int32)]),
E       max_line_width=1,
E       precision=1,
E       suppress_small=False,
E       fn_tree='ivy.functional.frontends.jax.numpy.array_str',
E       test_flags=FrontendFunctionTestFlags(
E           num_positional_args=0,
E           with_out=False,
E           inplace=False,
E           as_variable=[False],
E           native_arrays=[False],
E           generate_frontend_arrays=False,
E       ),
E   )
E

**Comment by Sarvesh-Kesharwani** [comment]
> E ivy.utils.exceptions.IvyException: <U27 is not supported by ivy E Falsifying example: test_jax_numpy_array_str( E frontend='jax', E on_device='cpu', E dtype_and_x=(['int32'], [array([0], dtype=int32)]), E max_line_width=1, E precision=1, E suppress_small=False, E fn_tree='ivy.functional.frontends.jax.numpy.array_str', E test_flags=FrontendFunctionTestFlags( E num_positional_args=0, E with_out=False, E inplace=False, E as_variable=[False], E native_arrays=[False], E generate_frontend_arrays=False, E ), E ) E

Hey Abdul, I have already mentioned this issue related to <U5 dtype above, which ivy's frontend_test function doesn't support or handle in any way,  I think this is a corner case, can you please tell  me how do i handle this?
Thanks!

**Comment by hmahmood24** [comment]
Hey @Sarvesh-Kesharwani. I'm not sure entirely for this function since this is not an array manipulation function as such and so we might not be able to test it properly with our current testing design. Could you please take this up with the testing team in Discord to see how we can approach this problem? Thanks

File: array_str_lab.py
Comment (abdulasiraj) [inline array_str_lab.py:1]: Oh, I think you forgot to remove testing files.

**Review by abdulasiraj — CHANGES_REQUESTED** [review]

**Review by abdulasiraj — CHANGES_REQUESTED** [review]
` Failed: [undefined]TypeError: test_frontend_function() missing 1 required keyword-only argument: 'backend_to_test'`

**Comment by danielmunioz** [comment]
Closing this as inactive, as it has been more than 2 weeks without any activity

**Pull request closed without merging**